### ISSUE TITLE
self-hosted, kodiakhq: status ⛴ attempting to merge PR (merging) 

### ISSUE THREAD

**Issue by haghabozorgi** [chdsbd/kodiak#401]
We are running kodiak in our kube cluster,  and when we test a test PR, it works as expected, it requires  the PR is merged, which has and the branch is deleted as expected, but the checks tab in github for that merged pr shows

```
kodiakhq: status ⛴ attempting to merge PR (merging) 
```

github checks tab snippet

![Screenshot_20200521_173058](https://user-images.githubusercontent.com/1186748/82618783-df27b380-9b88-11ea-8206-1d20440cfbbd.png)

github repo 'branch' settings

![Screenshot_20200521_173601](https://user-images.githubusercontent.com/1186748/82619073-c79cfa80-9b89-11ea-9a14-294a8984819d.png)

the .kodiak.toml

```
version = 1
[merge]
automerge_label = "auto-merge" # default: "automerge"
require_automerge_label = true
blacklist_title_regex = "" # default: "^WIP:.*", options: "" (disables regex), a regex string (e.g. ".*DONT\s*MERGE.*")
blacklist_labels = [] # default: [], options: list of label names (e.g. ["wip"])
method = "squash" # default: "merge", options: "merge", "squash", "rebase"
delete_branch_on_merge = true # default: false
block_on_reviews_requested = false # default: false
notify_on_conflict = true # default: true
optimistic_updates = true # default: true
dont_wait_on_status_checks = [] # default: [], options: list of check names (e.g. ["ci/circleci: lint_api"])
update_branch_immediately = false # default: false
prioritize_ready_to_merge = false # default: false
do_not_merge = false # default: false

[merge.message]
title = "github_default" # default: "github_default", options: "github_default", "pull_request_title"
body = "github_default" # default: "github_default", options: "github_default", "pull_request_body", "empty"
include_pr_number = true # default: true
body_type = "markdown" # default: "markdown", options: "plain_text", "markdown", "html"
strip_html_comments = false # default: false
```

I would expect that kodiakhq:status check to show completed. Please let me know what other info I can provide. Thanks for your efforts on this project.

**Comment by sbdchd** [comment]
Just to make sure I understand,  Kodiak is merging PRs but after merge, the status message on github is showing `status ⛴ attempting to merge PR (merging)` instead of something like `merge complete`?

Hmm seems like we don't set a status message on merge success: https://github.com/chdsbd/kodiak/runs/691144867

**Comment by haghabozorgi** [comment]
@sbdchd  thank you for the quick reply, yes that is exactly the issue.

**Comment by chdsbd** [comment]
This has been fixed by #469 which has now been deployed.

**Comment by haghabozorgi** [comment]
Thank you for the fix.
